release_notes/47-release-notes.adoc

```diff
@@ -478,4 +480,27 @@ This release provides fixes for the following security issues:
 * zlib: Out-of-bounds pointer arithmetic in `inftrees.c` link:https://access.redhat.com/security/cve/CVE-2016-9840[(CVE-2016-9840)]
 * krb5: Kerberos RC4-HMAC-MD5 checksum vulnerability link:https://access.redhat.com/security/cve/CVE-2025-3576[(CVE-2025-3576)]
 
+[id="about-release-475_{context}"]
+== About release version 4.7.5
+
+*Release date*: {ga-date-475}
+
+This release of {product-title-short} 4.7 provides the following bug fixes:
+
+//ROX-29320
+* Before this update, incorrect interpretation of Red Hat Enterprise Linux (RHEL) 10 Common Platform Enumeration (CPE) strings caused Scanner V4 to fail distribution checks on RHEL 10 systems. With this update, an updated RHEL CPE major version pattern resolves the issue, and Scanner V4 can now correctly support RHEL 10.
```

Comment:
I think we can simplify this a bit to remove some of the details (also I think it's a bit wrong as it's currently written):

How about:

> Before this update, Scanner V4 incorrectly claimed it did not support RHEL 10 even though it always did. Scanner V4 now correctly indicates that it does support RHEL 10.